docs/src/content/docs/commands/HGETALL.md

```diff
@@ -13,34 +13,43 @@ HGETALL key
 
 ## Parameters
 
-- `key`: The key of the hash from which you want to retrieve all fields and values. This parameter is mandatory.
+| Parameter       | Description                                      | Type    | Required |
+|-----------------|--------------------------------------------------|---------|----------|
+| `key`           | The key of the hash from which you want to retrieve all fields and values.                   | String  | Yes      |
 
-## Return Value
 
-The `HGETALL` command returns an array of strings in the form of field-value pairs. If the key does not exist, an empty array is returned.
+## Return values
+
+
+| Condition                                      | Return Value                                      |
+|------------------------------------------------|---------------------------------------------------|
+| The `key` exists and is a hash | Array of strings
+| The `key` does not exist           |  Empty array or set                                            |
```

Comment:
Hi, I am unable to find DiceDB's response for HGETALL with a non existent key saying "set". I am running latest master on my local. Please confirm.

Reply:
Resolved